src/modules/admin/dto/admin.dto.ts

```diff
@@ -24,10 +24,6 @@ export class RegisterUserRequest {
   @IsNotEmpty()
   username: string;
 
-  @ApiProperty({ example: 'org1', required: true })
```

Comment:
don't we need these?

Reply:
This is for registering a user. Admins should not define the org for the user they're registering to. By design, admin should be restricted to only add users in their own organisation.